app/models/notifications/nine/record_voucher_date_housing_subsidy_admin.rb

```diff
@@ -0,0 +1,23 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module Notifications::Nine
+  class RecordVoucherDateHousingSubsidyAdmin < ::Notifications::Base
+    def self.create_for_match! match
+      match.housing_subsidy_admin_contacts.each do |contact|
+        create! match: match, recipient: contact
+      end
+    end
+
+    def decision
+      match.nine_approve_match_housing_subsidy_admin_decision
```

Comment:
This doesn't feel like the right decision, but I could be wrong.  Shouldn't this be something to do with the housed date? (same with the label below)